src/State/AuthFlow/api.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+// TODO: load from the env config once implemented
+const COGNITO_URL =
+  'https://e6b8r4dht6.execute-api.us-west-2.amazonaws.com/dev';
+
+export function authenticate(token) {
+  const fetchBody = {
+    token: 'SAMLResponse=' + token,
+    AccessKeyId: null,
+    EmpEmail: null,
+    UserKey: null,
+    SessionToken: null,
+    SamlHash: null,
+    SecretKey: null
+  };
+  return axios
+    .post(`${COGNITO_URL}/authenticate`, fetchBody)
+    .then(response => {
+      return response.data;
+    })
+    .then(error => error);
```

Comment:
Should this be `catch()`? (I might be wrong because it's been a bit since axios). 

In either case, has this been verified to get caught in the saga if there is an error?

Reply:
I remember reading something about `.then` vs `.catch` when using redux sagas, but will verify if the FAILURE action is triggered